bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -47,89 +48,87 @@ __program__ = "pycbc_pygrb_plot_chisq_veto"
-                                                     chisq_threshold=opts.newsnr_threshold,
-                                                     null_grad_thresh=opts.null_grad_thresh,
-                                                     null_grad_val=opts.null_grad_val)
-            data[snr_type] = data[snr_type][data['reweighted_snr'] != 0]
-            data[veto_type] = data[veto_type][data['reweighted_snr'] != 0]
-        # Gather data on degrees of freedom
-        for cs in ['', 'bank_', 'cont_']:
-            dof = cs+'chisq_dof'
-            data[dof] = numpy.unique(trigs_or_injs.get_column(dof))
+        trigs_or_injs = ppu.load_triggers(input_file, vetoes)
+        
+        num_trigs_or_injs = len(trigs_or_injs['network/end_time_gc'][:])
+        
+        if snr_type in ['coherent', 'null', 'reweighted']:
+            data[snr_type] = trigs_or_injs['network/%s_snr' % snr_type][:]
+        elif snr_type == 'single':
+            att = opts.ifo[0].lower()
+            data[snr_type] = trigs_or_injs['%s/snr_%s' % (opts.ifo, att)][:]
+
+        # Calculate coincident SNR
+        elif snr_type == 'coincident':
+            
+            # Obtain all single SNRs
+            single_snr = {}
+            for ifo in ifos:
+                ifo_att = {'G1': 'g', 'H1': 'h1', 'H2': 'h2', 'L1': 'l',
+                           'V1': 'v', 'T1': 't'}
+                att = ifo_att[ifo]
+                single_snr[ifo] = trigs_or_injs['%s/snr_%s' % (ifo, att)][:]
+            
+            # Calculate coincident SNR
+            if len(ifos) > 1:
+                # Initialize some dictionaries we will use
+                single_snr_sq = dict((ifo, None) for ifo in ifos)
+                snr_sum_square = numpy.zeros(num_trigs_or_injs)
+                for ifo in ifos:
+                    # First square the individual SNR's
+                    single_snr_sq[ifo] = numpy.square(single_snr[ifo])
+                    # Then add them
+                    snr_sum_square = numpy.add(snr_sum_square,
+                                               single_snr_sq[ifo])
+                # Now obtain the square root
+                data[snr_type] = numpy.sqrt(snr_sum_square)
+
+        # Tags to find vetoes in HDF files
+        veto_tags = {'standard': 'chisq',
+                     'bank': 'bank_chisq',
+                     'auto': 'cont_chisq'}
+
+        data[veto_type] = \
+            trigs_or_injs['%s/%s' % (opts.ifo, veto_tags[veto_type])][:]
+        data['dof'] = numpy.unique(
+            trigs_or_injs['%s/%s_dof' % (opts.ifo, veto_tags[veto_type])][:])
+
+        logging.info("%d triggers found.", num_trigs_or_injs)
 
     return data
 
+# Format single detector chi-square data as numpy array and floor at 0.005
+def format_single_chisqs(trig_ifo_cs, ifos):
+    """Format single IFO chi-square data as numpy array and floor at 0.005"""
+
+    for ifo in ifos:
+        trig_ifo_cs[ifo] = numpy.asarray(trig_ifo_cs[ifo])
```

Comment:
This is no longer necessary.  Remove the line and update the help message and the comment above the `def` accordingly.